README.md

```diff
@@ -2,57 +2,79 @@
 ![Version](https://badge.fury.io/py/lumigo-tracer.svg)
 ![codecov](https://codecov.io/gh/lumigo-io/python_tracer/branch/master/graph/badge.svg?token=6EgXIlefwG)
 
+This is the Python version of Lumigo's tracer library.
+
 Supported Runtimes: 3.6, 3.7, 3.8
 
-# How To Use
-## With Lambda Layers:
-* Use the latest ARN version [from these tables](https://github.com/lumigo-io/python_tracer/tree/master/layers)
-## With Serverless framework:
-* Install the [**serverless-lumigo-plugin**](https://github.com/lumigo-io/serverless-lumigo-plugin/blob/master/README.md)
-## Manually
-* install with `pip install lumigo_tracer` <br/> 
-* import using `from lumigo_tracer import lumigo_tracer`
-* wrap you lambda function using `@lumigo_tracer` or `@lumigo_tracer(token='XXX')`. As an example, your lambda should look like: 
-```
+# Using the tracer
+There are three ways to use the tracer with your AWS Lambda functions: using our Lambda Layer, using the Serverless framework, or instrumenting your functions manually.
+
+## Lambda layers
+Using our Lambda layer lets you automatically instrument your Lambda functions. Simply use the latest ARN [from these tables](https://github.com/lumigo-io/python_tracer/tree/master/layers)
```

Comment:
You are right about 3 ways to instrument - manually, plugin and auto trace through the ui. Maybe you should include a link to the ui autotracing https://docs.lumigo.io/docs/auto-instrumentation

Reply:
Added a link to the docs!